vars/essentialsTest.groovy

```diff
@@ -26,7 +26,7 @@ def call(Map params = [:]) {
             if (configData.ath != null && !configData.ath.disabled) {
                 stage("Run ATH") {
                     dir("ath") {
-                        runATH jenkins: customWarURI, metadataFile: metadataPath, javaOptions: ["-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn", testPluginResolution]
+                        runATH jenkins: customWarURI, metadataFile: metadataPath, javaOptions: ['--no-transfer-progress', testPluginResolution]
```

Comment:
and this

Reply:
tbh can't see anything using essentials test

Reply:
Possibly unused. IIRC some sort of system planned by @oleg-nenashev and/or @carlossg but never widely adopted. Someone could check for known usages in trunk sources.

Reply:
https://github.com/jenkinsci/jep/blob/master/jep/309/README.adoc FTR